charts/sourcegraph/templates/precise-code-intel/worker.Deployment.yaml

```diff
@@ -28,6 +28,7 @@ spec:
     metadata:
       annotations:
         kubectl.kubernetes.io/default-container: precise-code-intel-worker
+        checksum/config: {{ include (print $.Template.BasePath "/minio/minio.Secret.yaml") . | sha256sum }}
```

Comment:
I would probably rename the annotation key to something more specific, such as `checksum/minio-secret`,  in case we need more checksum in the future.

Reply:
👍  I started to swap it in the db PR but didn't make the same change here, will update.